pattern-filter/solutions/pattern-filter/pom.xml

```diff
@@ -0,0 +1,110 @@
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>com.redhat.training</groupId>
+	<artifactId>transform-messages</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>transform-messages</name>
+	<description>GE: Generating Hello World Messages with the Scheduler Component</description>
+	<properties>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+		<maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
+		<fuse.version>7.9.0.fuse-sb2-790065-redhat-00001</fuse.version>
+	</properties>
+	<dependencyManagement>
+		<dependencies>
+		  <dependency>
+			<groupId>org.jboss.redhat-fuse</groupId>
+			<artifactId>fuse-springboot-bom</artifactId>
+			<version>${fuse.version}</version>
+			<type>pom</type>
+			<scope>import</scope>
+		  </dependency>
+		</dependencies>
+	  </dependencyManagement>
+	<dependencies>
+		<!-- camel dependencies -->
+		<dependency>
+			<groupId>org.apache.camel</groupId>
+			<artifactId>camel-spring-boot-starter</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-test</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.camel</groupId>
+			<artifactId>camel-test-spring</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<!-- JMS dependencies -->
+		<dependency>
+    		<groupId>org.apache.activemq</groupId>
+    		<artifactId>artemis-jms-client</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.camel</groupId>
+			<artifactId>camel-jms-starter</artifactId>
+		</dependency>
+		<!-- TODO Add jaxb dependency here -->
+		<dependency>
+			<groupId>org.apache.camel</groupId>
+			<artifactId>camel-jaxb</artifactId>
+		</dependency>
+		<!-- TODO Add camel-xmljson and xom dependencies here -->
+		<dependency>
+  			<groupId>org.apache.camel</groupId>
+  			<artifactId>camel-xmljson</artifactId>
+		</dependency>
+		<dependency>
+  			<groupId>xom</groupId>
+  			<artifactId>xom</artifactId>
+  			<version>1.3.7</version>
+		</dependency>
+		<!-- TODO Add camel-jsonpath dependency here -->
+		<dependency>
+  			<groupId>org.apache.camel</groupId>
+  			<artifactId>camel-jsonpath</artifactId>
+		</dependency>
+	</dependencies>
+	<repositories>
+		<repository>
+		  <id>red-hat-ga-repository</id>
+		  <url>https://maven.repository.redhat.com/ga</url>
+		</repository>
+	  </repositories>
+	  <pluginRepositories>
+		<pluginRepository>
+		  <id>red-hat-ga-repository</id>
+		  <url>https://maven.repository.redhat.com/ga</url>
+		</pluginRepository>
+	  </pluginRepositories>
+	<build>
+		<plugins>
+			<plugin>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<version>${maven-compiler-plugin.version}</version>
+				<configuration>
+					<source>1.8</source>
+					<target>1.8</target>
```

Comment:
We opened a ticket to be sure that we target Java 11 so, now that we didn't commited this PR we can change this. This comment also applies to the other POM file.